appraisal.lic

```diff
@@ -25,6 +26,42 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
     true
   end
+  
+  def train_appraisal_with_pouches?
+    settings = get_settings
+    return false unless settings.train_appraisal_with_pouches
+    full_pouch_container = settings.full_pouch_container
+    if !full_pouch_container
+      echo "Must set full_pouch_container in YAML"
+      return false
+    end      
+    if full_pouch_container == settings.spare_gem_pouch_container
+      echo "*** Cannot have full pouches be stored where spare pouches are ***"
+      return false
+    end
+    
+    pouches = rummage('C pouch', full_pouch_container)
+    pouches.each_with_index do |p, i|
+      if i >= $ORDINALS.size
+        return true
+      end
+      
+      case bput("get #{$ORDINALS[i]} #{p} from my #{full_pouch_container}", '^You get ', '^What were you referring')
+      when /^You get /
+        bput("appraise #{p} quick", 'Roundtime')
+        waitrt?
+        fput("put my #{p} in my #{full_pouch_container}")
+      else
+        return true
+      end
+      
+      if DRSkill.getxp('Appraisal') >= 32
```

Comment:
Since you end the method with a true, you can change all of these returns into breaks and combine them. something like
```
break if DRSkill.getxp('Appraisal') >= 32
```